### PR TITLE
Try to fix Appveyor Python

### LINKED ISSUE #3088: MSVC Unvanquished CI is broken, maybe Appveyor updated some python
See: https://ci.appveyor.com/project/UnvanquishedDevelopment/unvanquished/builds/50559706

```
-- Found Python: C:/Python313/python.exe (found version "3.13.0") found components: Interpreter
-- Using CBSE Python executable: C:/Python313/python.exe
Traceback (most recent call last):
  File "<string>", line 1, in <module>
    import jinja2, yaml, collections, argparse, sys, os.path, re
ModuleNotFoundError: No module named 'jinja2'
CMake Error at tools/cbse/CBSE.cmake:48 (message):
  Missing dependences for CBSE generation.  Please ensure you have python ≥
  2, python-yaml, and python-jinja installed.
                               Use pip install -r tools/cbse/requirements.txt to install
Call Stack (most recent call first):
  CMakeLists.txt:173 (CBSE)
-- Configuring incomplete, errors occurred!
```

### PR DESCRIPTION
**Pull request by slipher**
Fixes #3088 (?)

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -40,12 +40,16 @@ before_build:
     --sub-ref=%APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH%:has=/sync$
     --print
   - python --version
-  - pip install -r tools/cbse/requirements.txt --upgrade
+  - python -m pip install -r tools/cbse/requirements.txt --upgrade
```
Comment (DolceTriade) [inline .appveyor.yml:43]: Let's leave comments for why things like pip don't work.

File: .appveyor.yml [hunk 2]
```diff
@@ -40,12 +40,16 @@ before_build:
...
 
 build_script:
   - cmd: cmake --version
   - cmd: call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars32.bat"
+  # We don't need iteration; the for loop executes only once. But the function for resolving
+  # something in PATH (like 'which') is in the FOR command
+  - cmd: FOR %%p IN (python.exe) DO set firstPython=%%~$PATH:p
```
Comment (DolceTriade) [inline .appveyor.yml:50]: Should we be looking for python3?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -28,22 +28,22 @@ strategy:
...
-      CMAKE_OPTIONS: '-G Ninja -DDAEMON_CBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=gcc-8 -DCMAKE_CXX_COMPILER=g++-8 -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
+      CMAKE_OPTIONS: '-G Ninja -DCBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=gcc-8 -DCMAKE_CXX_COMPILER=g++-8 -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
       BUILDER_OPTIONS: '-j$(nproc) -k 10'
     Linux Clang Native DLL:
       VM_IMAGE: 'ubuntu-20.04'
       INSTALL_DEPS: 'sudo apt-get update && sudo apt-get -y -q --no-install-recommends install libfreetype6-dev liblua5.2-dev python-yaml python-jinja2 ninja-build'
-      CMAKE_OPTIONS: '-G Ninja -DDAEMON_CBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
+      CMAKE_OPTIONS: '-G Ninja -DCBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
```
Comment (DolceTriade) [inline azure-pipelines.yml:36]: Python2 is deprecated, unsupported, and dead. We should not test it and should not care if we break it.

File: .appveyor.yml (hunk 1 above)
Comment (slipher) [reply .appveyor.yml:43]: It does work fine here (at the moment) but this is just more reliable. I added details in the commit message.

File: .appveyor.yml (hunk 2 above)
Comment (slipher) [reply .appveyor.yml:50]: There's not usually any "python3.exe" available on Windows. The problem we face is two dueling Python 3 installations

**Review by DolceTriade — APPROVED** [review]
appears to build

**Pull request merged**